src/main/resources/fine_tuning/models.yaml

```diff
@@ -0,0 +1,48 @@
+---
+models:
+  axiom:
+    name: axiom
+    fixed_params: "-axiom.n 30 -axiom.deterministic"
+    nexus_params:
+      rerankCutoff: "axiom.r"
+    params:
+      axiom.r:
+        type: int
+        lower: 1
+        upper: 50
+        pace: 1
+      axiom.top:
+        type: int
+        lower: 1
+        upper: 50
+        pace: 1
+      axiom.beta:
+        type: float
+        lower: 0
+        upper: 1
+        pace: 0.1
+    expected:
+      robust04:
+        map: 0.2532
+  rm3:
+    name: rm3
+    fixed_params: ""
+    params:
+      rm3.fbTerms:
+        type: int
+        lower: 1
+        upper: 50
+        pace: 1
+      rm3.fbDocs:
+        type: int
+        lower: 1
+        upper: 50
+        pace: 1
+      rm3.originalQueryWeight:
+        type: float
+        lower: 0
+        upper: 1
+        pace: 0.1
+    expected:
```

Comment:
What does `expected` mean? Do we want to be more specific?

Reply:
I think we can temporarily remove it - it was used in other regression tests where you put the expected values so that the regression script can compare the regression values with expected ones. If they are not equal the script will fail.

Reply:
`rerankCutoff`？ I think it can also be used for rm3？